custom_components/tahoma/__init__.py

```diff
@@ -7,15 +7,16 @@
 from pyhoma.client import TahomaClient
 from pyhoma.exceptions import BadCredentialsException, TooManyRequestsException
 
+from homeassistant.components.scene import DOMAIN as SCENE
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, EVENT_HOMEASSISTANT_STOP
 from homeassistant.core import HomeAssistant
 
-from .const import DOMAIN, SUPPORTED_PLATFORMS, TAHOMA_TYPES
+from .const import DOMAIN, TAHOMA_TYPES
```

Comment:
Why is `SUPPORTED_PLATFORMS` removed? Would be good to keep it, instead of just relying on the platforms defined in const.py.

Reply:
The supported platforms are including in TAHOMA_TYPES, and obviously we support all of them. So this [condition](https://github.com/iMicknl/ha-tahoma/pull/198/files#diff-d28fdb4b71ecdf95d6ca1c2a918de26bL73) does not make sense, never we will map a device we don't support.

Reply:
I like the simplified code a lot more 👍. My only concern was that if we move to core, we need to carefully check if there are no extra mappings for integrations not yet present in those PRs.

Reply:
>we need to carefully check if there are no extra mappings for integrations not yet present in those PRs. 

I don't get it.

Reply:
@tetienne a PR to core will not include all platforms, thus we need to be careful with the list containing the mapping.

Reply:
Oh I see. My goal is to clean as much as possible our code to make it more readable and much easier to review.